quit/conf.py

```diff
@@ -25,14 +26,16 @@ class Feature:
 
 
 class QuitConfiguration:
-    """A class that keeps track of the relation between named graphs and files."""
```

Comment:
Why do we still need this class? What does it do? Please add some documentation string.

Reply:
This was a (very) small superclass. I removed it.